webextensions/api/privacy.json

```diff
@@ -473,7 +473,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤58"
+                  "version_added": "21"
```

Comment:
✅  https://github.com/chromium/chromium/commit/cc3a5c8ca42579c0f9f710c18f51c566b4c75e86 landed in [20.0.1133.0](https://github.com/chromium/chromium/releases/tag/20.0.1133.0) AFTER 20.0.1132.